src/containers/Tenant/Diagnostics/Overview/TableInfo/prepareTableInfo.tsx

```diff
@@ -200,8 +213,15 @@ export const prepareTableInfo = (data?: TEvDescribeSchemeResult, type?: EPathTyp
         }),
     ];
 
-    //@ts-expect-error
-    const tabletMetricsInfo = formatObject(formatTabletMetricsItem, TabletMetrics);
+    const tabletMetricsInfo = formatObject(
+        formatTabletMetricsItem,
+        omit(TabletMetrics, [
+            'GroupReadIops',
+            'GroupReadThroughput',
+            'GroupWriteIops',
+            'GroupWriteThroughput',
```

Comment:
Removed fields, that we don't display, but they are not processed properly by TS